graphics/vpype/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vpype
+version             1.9.0
```

Comment:
Besides `version`, Portfiles should also specify `revision`, set to 0 by default:


```
...
version             1.9.0
revision            0
```

`revision` is incremented when a change is being made to port but its version is still the same.

Reply:
OK, will do.